### PR TITLE
Multiple defs

### PR DESCRIPTION
**Pull request by higs4281**
Adds support for multiple definitions and stops the brain load cycling.
Addresses #6  and #7

### CODE REVIEW

**Comment by higs4281** [comment]
@contolini @hillaryj @djdrisco what say you?

File: src/acrogov.coffee [hunk 1]
```diff
@@ -39,18 +42,35 @@ class AcroBot
         @loadPublicAcronyms()
         console.log("loaded public acronyms")
```
Comment (contolini) [inline src/acrogov.coffee]: Change all the `console.log()` statements to `@robot.logger.debug()` to reduce console noise. They'll be printed when `HUBOT_LOG_LEVEL="debug"`.

**Comment by higs4281** [comment]
@contolini gone

**Comment by contolini** [comment]
:+1:

I just added a bunch of tests. I'll push 'em up shortly.

**Comment by contolini** [comment]
@higs4281 pull down master to ensure you're up-to-date and run `npm version [major|minor|patch]` (choose one, your call) followed by `npm publish` to publish these changes to npm.

Then in chat do `cfpbot update`, wait a couple minutes for it to update its npm dependencies, and then `cfpbot die`.

Nice work with this update btw!

**Comment by higs4281** [comment]
@contolini those tests are great. i should not be rewarded for being facetious.

**Comment by contolini** [comment]
@higs4281 Don't flatter yourself! I was 90% done with the tests before I read your facetiousness. I figured out the good Hubot test structure while building https://github.com/catops/generator-catops-script.

**Pull request merged**